netlink.go

```diff
@@ -449,6 +449,8 @@ func assembleDestination(attrs []syscall.NetlinkRouteAttr) (*Destination, error)
 			d.ActiveConnections = int(native.Uint16(attr.Value))
 		case ipvsDestAttrInactiveConnections:
 			d.InactiveConnections = int(native.Uint16(attr.Value))
+		case ipvsDestAttrPersistentConnections:
+			d.PersistentConnections = int(native.Uint16(attr.Value))
```

Comment:
Looking at the kernel implementation this should be uint32?

Reply:
Yeap, the connection attributes `ActiveConnections` `InactiveConnections` `PersistentConnections` also should be uint32.  Unified could be better.